gnc/sub8_perception/include/sub8_pcl/buoy.hpp

```diff
@@ -0,0 +1,84 @@
+#pragma once
+#include <string>
+#include <vector>
+#include <iostream>
+#include <pcl/console/time.h>  // TicToc
+#include <pcl/common/time.h>
+#include <pcl/console/print.h>
+#include <pcl/visualization/pcl_visualizer.h>
+#include <pcl/conversions.h>
+#include <pcl/PCLPointCloud2.h>
+#include <pcl/filters/statistical_outlier_removal.h>
+#include <pcl_conversions/pcl_conversions.h>
+
+#include <algorithm>
+#include <opencv2/opencv.hpp>
+#include <cv_bridge/cv_bridge.h>
+#include <image_transport/image_transport.h>
+#include <image_geometry/pinhole_camera_model.h>
+
+#include <tf/transform_listener.h>
+#include <sensor_msgs/image_encodings.h>
+
+#include <eigen_conversions/eigen_msg.h>
+#include <Eigen/StdVector>
+#include <sub8_pcl/pcl_tools.hpp>
+#include <sub8_pcl/cv_tools.hpp>
+
+#include "ros/ros.h"
+
+#include "sub8_msgs/VisionRequest.h"
+
+// For stack-tracing on seg-fault
+#define BACKWARD_HAS_BFD 1
+#include <sub8_build_tools/backward.hpp>
```

Comment:
I'm not sure how it's supposed to work but I haven't had anything interesting happen after a seg fault.